sass/directives/_card_blocks.scss

```diff
@@ -0,0 +1,66 @@
+@mixin card-block($include_strip: true) {
+  @include card--tile;
+  @include transform(scale(1)); // TODO mixin that includes transition and transform
+  @include transition(transform 0.4s ease-in);
+
+  padding-top: calc(#{$base-spacing-medium} + #{$small-strip-height});
+  color: $body-text-color;
+  text-align: center;
+  margin: $base-spacing-medium / 2;
+
+  h3 {
+    text-transform: uppercase;
+    font-weight: $font-weight-bold;
+    font-size: $small-font-size;
+    margin-top: $base-spacing-small;
+  }
+
+  &:before {
+    @if $include_strip {
+      @include bs__small-strip;
+    }
+  }
+
+  @media only screen and (min-width: $small-screen-max) { // TODO turning off the fanciness on mobile but should there still be some sort of state for usability?
+    &:hover {
+      @include transform(scale(1.06));
+      @include transition(transform 550ms ease-out);
+
+      &:before {
+        background-color: $anchor-blue;
+      }
+    }
+  }
+}
+
+@mixin card-block__logo {
```

Comment:
I like this - my only problem with it is that it would apply these styles to any image within the card. I've implemented it for now, but we may want to revisit it one day. I'm still of the pursuasion that we'd want to apply this mixin directly to a semantic class just for the image tag, for maximum control.

However: the original use case I created for this mixin is a bit silly. There's no reason to create a mixin that takes in a single parameter (height) and applies a single style - that's reinventing the wheel. To this end, I would remove `height` from this mixin and apply it directly to the semantic class (and keep the mixin to apply the `margin-bottom`). Since we won't be using a semantic class for the image in this particular implementation, however, I'll continue taking in `height` as a parameter. Thanks!